moto/qldb/urls.py

```diff
@@ -0,0 +1,11 @@
+"""qldb base URL and path."""
+
+from .responses import QLDBResponse
+
+url_bases = [
+    r"https?://qldb\.(.+)\.amazonaws\.com",
+]
+
+url_paths = {
+    "0/.*$": QLDBResponse.dispatch,
```

Comment:
```suggestion
    "{0}/.*$": QLDBResponse.dispatch,
```

Typo - the key is converted into a regex to match the full url of the incoming request, with `{0}` acting as a substitute for the host (`qldb.amazonaws.com`), but it will not match anything as a literal `0`.

I figured this out by debugging the `BotocoreStubber` 
https://github.com/getmoto/moto/blob/52d2f06800f5c1db8b5fdb040e6537556ff5b45e/moto/core/botocore_stubber.py#L45

All requests are handled by that class, so debugging (or print-statements) can sometimes very quickly tell you why a request is not routed correctly. (In decorator-mode at least - server mode is a different story..)

Reply:
Looking back, it seems like this was generated by the scaffold command. Worthy of an issue?

Reply:
I raised https://github.com/getmoto/moto/pull/7981 to fix the scaffolding. Thank you for pointing that out!